image/tools/entrypoint.sh

```diff
@@ -11,6 +12,9 @@ while getopts "c:b:e:d:" opt; do
     c)
         component="$OPTARG"
         ;;
+    n)
```

Comment:
you need to add the arg `n:` in here: `getopts "c:b:e:d:"`

Reply:
whoops, I did do that, and forgot to push it